### ISSUE TITLE
`Uncaught ReferenceError: query is not defined` on v1.2.6

### ISSUE THREAD

**Issue by geelen** [isaac-mcfadyen/d1-console#4]
<img width="516" alt="image" src="https://user-images.githubusercontent.com/23264/201223337-fa21ef01-6ad6-4983-84d9-31cb9f5c0011.png">

Was trying to take a screenshot for the blog post 🙃

**Comment by isaac-mcfadyen** [comment]
Will be fixed by tonight 😄

**Comment by geelen** [comment]
Oh never mind, I'm on an old version, for some reason `npx d1-console` is giving me old stuff, even with `--prefer-online`

**Comment by geelen** [comment]
Ah but now I've lost my pretty colours for the screenshot! 

<img width="876" alt="image" src="https://user-images.githubusercontent.com/23264/201225309-79588f7c-b62c-4c5e-8240-18c4171990a3.png">

**Comment by geelen** [comment]
Also, again for the screenshot, I'd love if the only help lines printed by default were:

```
To list the available databases on your account, use SHOW DATABASES;
If you know the name of the database you would like to query, run USE <name>;
To see the full list of commands, type HELP;
```

Maybe I should just make a PR with my changes...

**Comment by isaac-mcfadyen** [comment]
I'm actually porting this to [`commander`](https://www.npmjs.com/package/commander) instead of my own (somewhat hacky) implementation. 🙂 

Should be done by tonight, and I'm going to be moving some of the `USE/SHOW` commands to flags/arguments as part of that. I'll also add colors & less of an info-dump at launch 😄

**Comment by geelen** [comment]
Beautiful! Let me know when I should retry and take screenshots.

**Comment by isaac-mcfadyen** [comment]
@geelen Done! This has now been shipped in `1.4.0` 🎉
